src/common/libjob/policy.c

```diff
@@ -0,0 +1,360 @@
+/************************************************************\
+ * Copyright 2020 Lawrence Livermore National Security, LLC
```

Comment:
Also, I'm assuming you kept 2020 copyright date here on purpose due to previous copyright date of code (Note: `git blame` shows the actual date of inclusion of the moved code is 2022, but I doubt this matters much!)

Reply:
Since you took the trouble to look up the actual date the code was created, I'll change it to 2022 - thanks!